cime_config/namelist_definition_drv_flds.xml

```diff
@@ -145,4 +145,20 @@
     </desc>
   </entry>
 
+  <!-- ========================================================================================  -->
+  <!-- Ozone control                                                                              -->
+  <!-- ========================================================================================  -->
+
+  <entry id="atm_ozone_frequency">
+    <type>char</type>
+    <category>ozone_coupling</category>
+    <group>ozone_coupling_nl</group>
+    <desc>
+      Frequency of surface ozone field passed from CAM to surface components.
+      Surface ozone is passed every coupling interval, but this namelist flag
+      indicates whether the timestep-level values are interpolated from a
+      coarser temporal resolution.
+    </desc>
+  </entry>
```

Comment:
What are the excepted settings for this option?

Reply:
I went back and forth on whether to include valid values here. I tentatively decided not to list them here because of maintainability concerns: Nothing in this repository cares what the value of this variable is, since the code to read it in is in the CESM_share repository (https://github.com/ESCOMP/CESM_share/pull/4). So I felt that listing the valid values here might be more painful than helpful, because if a new possible value is added, then this repository would need to be updated just for the sake of adding this new value to the valid_values list. Note that I *did* define the valid values in CAM's namelist_definition; there they serve the purpose of ensuring that any entries in user_nl_cam are valid.

However, I don't feel strongly on that point, so if you feel that it would be valuable to add the valid_values here, I'm happy to do so.

Reply:
I am fine with not listing valid values here.

Reply:
For now I agree with @fvitt.